tests/phpunit/integration/ContextTest.php

```diff
@@ -230,4 +230,8 @@ function() {
 		$context = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
 		$this->assertTrue( $context->is_network_active() );
 	}
+
+	public function test_get_current_entity() {
+		$this->markTestSkipped( 'Test not written yet.' );
```

Comment:
Shockingly I haven't had to run the PHP tests locally in the entire length of working on this project, but I wanted to add some here for the new method I created. @aaemnnosttv helped me out with getting them running locally but I didn't have time to get through it tonight. I'll try to write these tomorrow.